webapp/src/components/messages/versionMessage.tsx

```diff
@@ -88,4 +88,7 @@ const VersionMessage = React.memo(() => {
         </div>
     )
 })
+
+VersionMessage.displayName = 'VersionMessage'
```

Comment:
I'm not sure what this line is doing.

Reply:
This was added becuase of the linter error, having the displayName property will not create any issue, instead, it will help in debugging and displaying the component names in the error stack trace.

Reply:
The `displayName` is also set automatically if we define the component as `function MyComponent`, although it's not set if you define it as `const MyComponent = ...`. I'm not sure how to have `React.memo` set it automatically, but maybe it'll do that if you were to write this as `const VersionMessage = React.memo(function VersionMessage(){...});`